### PR TITLE
add patch for emacs with glibc-2.34

### PR DESCRIPTION
**Pull request by joequant**

### CODE REVIEW

File: var/spack/repos/builtin/packages/emacs/package.py [hunk 1]
```diff
@@ -61,6 +61,8 @@ class Emacs(AutotoolsPackage, GNUMirrorPackage):
     conflicts('@:26.3', when='platform=darwin os=catalina')
     conflicts('+native', when='@:27', msg="native compilation require @master")
 
+    patch('glibc-2.34.patch')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/emacs/package.py:64]: Can you copy the comment in the Gentoo patch over the directive? That will give people context by reading only `package.py`. Also is the patch to be applied unconditionally (i.e. is it valid for each version of emacs we support)?

**Review by alalazo — CHANGES_REQUESTED** [review]
One minor request and a question

File: var/spack/repos/builtin/packages/emacs/package.py (hunk 1 above)
Comment (bernhardkaindl) [reply var/spack/repos/builtin/packages/emacs/package.py:64]: @joequant ping for the question on the patch

**Comment by alalazo** [comment]
Closing, since there has been no activity in a while. Feel free to reopen if you want to continue work on this.

**Pull request closed without merging**